.gitmodules

```diff
@@ -56,3 +56,6 @@
 [submodule "third_party/nghttp2"]
 	path = third_party/nghttp2
 	url = https://github.com/tarantool/nghttp2.git
+[submodule "third_party/libunwind"]
+	path = third_party/libunwind
+	url = https://github.com/tarantool/libunwind.git
```

Comment:
Libunwind is builded in `static-build/CMakeLists.txt` as well. Now it look unnecessary (in the 'dead code' sense) and should be removed.

Reply:
Fixed: removed libunwind from static build.